coldfront/core/utils/views/mou_views.py

```diff
@@ -1,4 +1,5 @@
 import logging
+import os
```

Comment:
We had discussed creating the directory in the view, but I'm not seeing it here--has the approach changed?

Reply:
iirc i figured out that the directories are automatically created when saving a file